pytest_fixtures/component/smartproxy.py

```diff
@@ -22,13 +19,10 @@ def import_puppet_classes(default_smart_proxy):
 @pytest.fixture(scope='module')
 def module_fake_proxy(request, module_target_sat):
     """Create a Proxy and register the cleanup function"""
-    args = {'name': gen_string(str_type='alpha')}
```

Comment:
Earlier I found this issue on the new checout sat, try including `proxy_port_range` fixture if this fixes this issue.

Reply:
tried that out but somehow it didn't reach the destructive satellite, however adding the port enablement to the fake proxy fixture worked

Reply:
yes, its for default_sat, for destructive_sat, its better to be explicit for now